src/store/store.js

```diff
@@ -1,54 +1,28 @@
-import { _ } from "vue-underscore";
 import Vue from "vue";
 import Vuex from "vuex";
-import moviesJson from "./../assets/movies.json";
+import axiosApi from "./../axios";
 
 Vue.use(Vuex);
 
 export default new Vuex.Store({
     state: {
         searchText: "",
         searchOption: "title",
-        sortOption: "releaseDate",
-        movies: moviesJson,
+        sortOption: "release_date",
+        movies: [],
+        movieById: null,
     },
     getters: {
         getAllFilms: (state) => {
-            return _.sortBy(state.movies, state.sortOption).reverse();
+            return state.movies.sort((a, b) => b[state.sortOption] - a[state.sortOption]);
```

Comment:
The movies should be retrieved already sorted from that API, can't get why the sorting here is needed.